test/functoria/e2e/build.t

```diff
@@ -12,6 +12,7 @@ Build an application.
   dune
   dune.build
   dune.config
+  key.ml
```

Comment:
would you mind to elaborate what the newly generated `key.ml` is for? AFAICT it is "if you define runtime keys in config.ml", is this true? Shouldn't these keys be defined in unikernel.ml instead? My gut feeling is that the complexity increases if you generate more code, so I'd be eager to see less code being generated, and more being spelled out.

Reply:
I've moved the runtime keys in app.ml